.github/workflows/ci.yml

```diff
@@ -17,35 +17,38 @@ jobs:
 
   build:
     name: Build
-    runs-on: Ubuntu-20.04
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.1', '7.2', '7.3', '7.4', '8.0']
+        php: ['7.2', '7.3', '7.4', '8.0', '8.1']
```

Comment:
not related to this PR

Reply:
I've put the PHP 7.1 option back in place, so now this only adds 8.1 and switches the latest to it.